src/components/Disputes/DisputeDetail.js

```diff
@@ -10,19 +10,17 @@ import NoEvidence from './NoEvidence'
 
 const DisputeDetail = React.memo(function DisputeDetail({ match }) {
   const history = useHistory()
-  const disputes = useDisputesSubscription()
   const { id: disputeId } = match.params
 
-  const dispute = useMemo(
-    () => disputes.find(dispute => dispute.id === disputeId),
-    [disputeId, disputes]
+  const { dispute, fetching: disputeFetching } = useDisputeSubscription(
```

Comment:
I think that for subscriptions, fetching is always true 😄

Reply:
Not this one haha https://github.com/aragon/court-dashboard/pull/41/files#diff-4451f186da266821f8a6c366b69e3dc0R17

But I should probably use something else like `initialFetching` :thinking: